.github/workflows/backport.yml

```diff
@@ -15,15 +16,15 @@ jobs:
     steps:
       - name: GitHub App token
         id: github_app_token
-        uses: tibdex/github-app-token@v1.7.0
+        uses: tibdex/github-app-token@v1.5.0
```

Comment:
Why the downgrade?

Reply:
Wanted to keep this in sync with all our other repos. Also, I didn't see huge changes between both marketplace versions: https://github.com/tibdex/github-app-token/compare/v1.7.0...v1.5.0.